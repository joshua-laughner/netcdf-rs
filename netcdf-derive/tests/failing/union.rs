#[derive(netcdf_derive::NcType)]
union NoSupportedUnion {
    A: i32,
    B: u32,
}

fn main() {}
