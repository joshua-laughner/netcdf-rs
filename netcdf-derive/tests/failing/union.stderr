error: Can not derive NcType for union
 --> tests/failing/union.rs:2:1
  |
2 | / union NoSupportedUnion {
3 | |     A: i32,
4 | |     B: u32,
5 | | }
  | |_^
  |
  = note: netCDF has no concept of Union type
